lua/mapvote/server/modules/thumb.lua

```diff
@@ -18,6 +18,8 @@ net.Receive( "MapVote_RequestWorkshopIDTable", function( _, ply )
     for _, map in ipairs( requestedMaps ) do
         if MapVote.addonWorkshopIDs[map] then
             addonWorkshopIDs[map] = MapVote.addonWorkshopIDs[map]
+        else
+            print( "MapVote: No workshop ID found for map", map )
```

Comment:
Could this be an `ErrorNoHalt(WithStack)`?

Reply:
its not necessarily an error, fastdl maps would not have workshop ids